packages/nextjs/src/server/index.ts

```diff
@@ -1,3 +1,5 @@
+import { buildErrorThrower } from '@clerk/shared';
```

Comment:
```suggestion
import { buildErrorThrower } from '@clerk/shared/error';
```

Small optimization for tree-shaking